src/TwigComponent/doc/index.rst

```diff
@@ -840,86 +841,96 @@ The ``with`` data is what's mounted on the component object.
 .. note::
 
     Embedded components *cannot* currently be used with LiveComponents.
-    
+
 Component HTML Syntax
 ---------------------
-    
+
 .. versionadded:: 2.8
 
     This syntax was been introduced in 2.8 and is still experimental: it may change in the future.
-    
+
 Twig Components come with an HTML-like syntax to ease the readability of your template:
 
 .. code-block:: html+twig
 
-    <twig:Alert></:Alert>
+    <twig:Alert></twig:Alert>
     // or use a self-closing tag
-    <twig:Alert/>
-    
-You can pass props to your component by using HTML attributes. Suppose you have the following component:
+    <twig:Alert />
+
+Passing Props as HTML Attributes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Passing props is done with HTML attributes. For example if you have this component::
```

Comment:
```suggestion
Passing props is done with HTML attributes. For example if you have this component:
```

Reply:
@jdreesen actually, the double `::` is needed. We use RST format instead of Markdown. code blocks are declared as `.. code-block:: php`. But since we have so many PHP code blocks, it's boring to do that. That's why we use a RST trick to tell it that any code following two `::` is to be considered as PHP code.

Reply:
Oh, I didn't know that.